etna/core/middleware.py

```diff
@@ -0,0 +1,24 @@
+from datetime import datetime, timedelta
+from urllib.parse import quote
+
+from django.conf import settings
+
+
+class CookieMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        response = self.get_response(request)
```

Comment:
We should avoid doing anything when `FEATURE_COOKIE_BANNER_ENABLED` is `False`:

```suggestion
        response = self.get_response(request)
        if not settngs.FEATURE_COOKIE_BANNER_ENABLED:
            return response
```